src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -163,6 +163,48 @@ public ResponseEntity<Measure> updateMeasure(
     return response;
   }
 
+  @PutMapping("/measures/{id}/delete")
+  public ResponseEntity<Measure> deactivateMeasure(
```

Comment:
prettey much everythin in this method can be moved to service method `deactivateMeasure`

Reply:
I would recommend this be a `DELETE` mapping, since it will be our primary soft-delete for a measure